model/export/AssemblyExporterService.php

```diff
@@ -26,24 +26,56 @@
 use tao_helpers_Display;
 use tao_helpers_Export;
 use tao_helpers_File;
-use core_kernel_classes_Property;
 use core_kernel_classes_Resource;
 use common_ext_ExtensionsManager;
-use tao_models_classes_service_FileStorage;
 use tao_models_classes_service_ServiceCall;
 use tao_models_classes_export_RdfExporter;
 use oat\generis\model\OntologyAwareTrait;
 use oat\oatbox\log\LoggerAwareTrait;
 use oat\oatbox\service\ConfigurableService;
+use oat\tao\model\service\ServiceFileStorage;
+use oat\taoDeliveryRdf\model\assembly\AssemblyFilesReader;
+use oat\taoDeliveryRdf\model\assembly\AssemblyFilesReaderInterface;
 use oat\taoDeliveryRdf\model\DeliveryAssemblyService;
 
 class AssemblyExporterService extends ConfigurableService
 {
     use LoggerAwareTrait;
     use OntologyAwareTrait;
 
+    const SERVICE_ID = 'taoDeliveryRdf/AssemblyExporterService';
+    const OPTION_ASSEMBLY_FILES_READER = 'assembly_files_reader';
+    const OPTION_RDF_EXPORTER = 'rdf_exporter';
     const MANIFEST_FILE = 'manifest.json';
 
+    /**
+     * @var AssemblyFilesReaderInterface
+     */
+    private $assemblyFilesReader;
+
+    /**
+     * @var tao_models_classes_export_RdfExporter
+     */
+    private $rdfExporter;
+
+    /**
+     * AssemblyExporterService constructor.
+     * @param array $options
+     */
+    public function __construct($options = array())
+    {
+        parent::__construct($options);
+
+        if (!$this->getOption(self::OPTION_ASSEMBLY_FILES_READER) instanceof AssemblyFilesReaderInterface) {
+            throw new \InvalidArgumentException(sprintf('%s option value must be an instance of %s', self::OPTION_ASSEMBLY_FILES_READER, AssemblyFilesReaderInterface::class));

```

Comment:
declare the class with use statement, please

Reply:
Fixed